FluentUI/src/main/java/com/microsoft/fluentui/contextualcommandbar/CommandItem.kt

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.fluentui.contextualcommandbar
+
+import android.support.annotation.DrawableRes
+
+interface CommandItem {
+
+    @DrawableRes
+    fun getIcon(): Int
+
+    fun getContentDescription(): String? = null
```

Comment:
is this intended to be a used for accessibility?

Reply:
a related question, items will only be rendered as icons, so if there is no icon available the command button can not render (e.g. with a label instead)?
I ask as I have a scenario where commands can be added on the fly, so we may not have an icon to map the command to...

Reply:
> is this intended to be a used for accessibility?  

Yes
> a related question, items will only be rendered as icons, so if there is no icon available the command button can not render (e.g. with a label instead)?
I ask as I have a scenario where commands can be added on the fly, so we may not have an icon to map the command to...

Do you mean we need to add an option for display text instead of icon?

Reply:
Yes, we have cases where some buttons will not have an icon and we will want them showing just a localized string. This isn't currently possible, could it be added (priority is to show icon, if no icon show display text, if neither icon or display text do not show the button).
Currently if an icon is not provided an empty button is loaded, but probably it should not be?

Reply:
Supported text display

Reply:
Thank you, that works nicely for items with labels now